content/en/references/custom-ssl-certificates.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Custom SSL certificates
```

Comment:
Maybe we should rename this to "TLS" (SSL isn't really used anymore, same for the rest of the changes).

Reply:
I think SSL is still a common term for these but I've updated to TLS anyway. Thanks!